test/categories/test_electronic.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.electronic import Electronic
+import random
+
+
+class TestElectronic(unittest.TestCase):
```

Comment:
If you make sure that you have the latest changes from the original repo, the way to write unit tests has been simplified. See any other test class for reference